banyan/resource_service_web.go

```diff
@@ -486,6 +501,15 @@ func expandBackendWhitelist(d *schema.ResourceData) []string {
 	return items
 }
 
+func expandTLSSNIs(d *schema.ResourceData) []string {
```

Comment:
@rohitSangamnerkar default TLSSNI should continue to have "domain" as the default behaviour, if TLSSNI is specified it should be set as is.

Reply:
Updated code use domain as TLSSNI if TLSSNI is not specified. If it is specified existing TLSSNI will be replaced by specified TLSSNI.